Source/WebCore/page/PointerCaptureController.cpp

```diff
@@ -379,6 +379,14 @@ void PointerCaptureController::dispatchEventForTouchAtIndex(EventTarget& target,
 }
 #endif // ENABLE(TOUCH_EVENTS) && (PLATFORM(IOS_FAMILY) || PLATFORM(WPE))
 
+void PointerCaptureController::clearUnmatchedMouseDown(PointerID pointerID)
+{
+    if (RefPtr capturingData = m_activePointerIdsToCapturingData.get(pointerID)) {
+        if (capturingData->pointerIsPressed)
+            capturingData->pointerIsPressed = false;
+    }
```

Comment:
You could set this unconditionally, not like it will change anything for the false case.
```suggestion
    if (RefPtr capturingData = m_activePointerIdsToCapturingData.get(pointerID))
        capturingData->pointerIsPressed = false;
```